README.md

```diff
@@ -1 +1,83 @@
-# CytoPipe
\ No newline at end of file
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [CytoPipe](#cytopipe)
+- [Formatting](#formatting)
+  - [smkfmt](#smkfmt)
+- [Testing](#testing)
+  - [Dry Runs](#dry-runs)
+
+# CytoPipe
+
+# Formatting
```

Comment:
I think all headings after the first `# CytoPipe` should be `H2` headings:

AKA `## Formatting`

```suggestion
## Formatting
```

Reply:
Please fix all headings below as follows